include/wallaby/aruco.hpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * arhco.h
+ *
+ * Created on April 4, 2017
+ *      Author: Ryan Owens
+ */
+#ifndef INCLUDE_WALLABY_ARUCO_HPP_
+#define INCLUDE_WALLABY_ARUCO_HPP_
+
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <opencv2/aruco.hpp>
+#include <opencv2/calib3d.hpp>
+#include <opencv2/core.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/videoio.hpp>
+#include <string>
+#include <unistd.h>
+
+#include "wallaby/camera.hpp"
+
+namespace aruco {
+  class Aruco;
+  static Aruco* instance;
```

Comment:
We usually hide this instance variable in the ``Aruco::instance()`` function like in
https://github.com/kipr/libwallaby/blob/master/src/wallaby_p.cpp#L114-L118